shared/wallets/send-form/body/container.js

```diff
@@ -1,11 +1,21 @@
 // @flow
 import Body from '.'
 import {compose, connect, setDisplayName, type TypedState} from '../../../util/container'
+import {bannerLevelToBackground} from '../../../constants/wallets'
 
-const mapStateToProps = (state: TypedState) => ({})
+const mapStateToProps = (state: TypedState) => ({
+  banners: (state.wallets.builtPayment.banners || []).map(banner => ({
```

Comment:
likely better to do this in mergeProps so you don't thrash (even with the new empty array each time)

Reply:
Fixed via 4ba6b67